<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BusinessManagerUI</class>
 <widget class="QWidget" name="BusinessManagerUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>客户管理</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="MyTableView" name="clientView"/>
         </item>
         <item>
          <widget class="SqlPageControleUI" name="clientPageCtrl" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="refleshBtn">
           <property name="text">
            <string>刷新</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addClientBtn">
           <property name="text">
            <string>添加客户</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clientEditBtn">
           <property name="text">
            <string>编辑客户</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="EditSaleManBtn">
           <property name="text">
            <string>指派业务员</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>导入客户</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>订单管理</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="MyTableView" name="orderView"/>
       </item>
       <item>
        <widget class="SqlPageControleUI" name="orderPageCtrl" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTableView</class>
   <extends>QTableView</extends>
   <header>mytableview.h</header>
  </customwidget>
  <customwidget>
   <class>SqlPageControleUI</class>
   <extends>QWidget</extends>
   <header>../sqlPageControleUI/sqlpagecontroleui.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
